cmd/root.go

```diff
@@ -152,6 +152,15 @@ func Run(c *cobra.Command, names []string) {
 	enableMetricsAPI, _ := c.PersistentFlags().GetBool("http-api-metrics")
 	unblockHTTPAPI, _ := c.PersistentFlags().GetBool("http-api-periodic-polls")
 	apiToken, _ := c.PersistentFlags().GetString("http-api-token")
+	healthCheck, _ := c.PersistentFlags().GetBool("health-check")
+
+	if healthCheck {
+		// health check should not have pid 1
+		if os.Getpid() == 1 {
+			os.Exit(1)
```

Comment:
If someone accidentally adds this to the command line, we should probably tell them the problem and prevent `restart: always` from pinning the CPU.

```suggestion
			time.Sleep(1 * time.Second)
			log.Fatal("The health check flag should never be passed to the main watchtower container process")
```

It would of course seem like this would also trigger if the container is up, but the main watchtower process has died, but this can never happen since that would stop the container.